liberum_core/src/node/store.rs

```diff
@@ -50,30 +58,76 @@ impl NodeStore {
             return Err(NodeStoreError::NodeDoesNotExist { name });
         }
 
-        let node = Node::load(&node_dir_path)
-            .await
-            .map_err(|_| NodeStoreError::LoadError { name })?;
+        if !node_dir_path.is_dir() {
+            error!(
+                dir_path = node_dir_path.display().to_string(),
+                "node dir path not a directory"
+            );
+
+            return Err(NodeStoreError::OtherError {
+                name,
+                err: anyhow!("node_dir_path is not a directory"),
+            });
+        }
 
-        Ok(node)
+        let config_path = node_dir_path.join(Node::CONFIG_FILE_NAME);
```

Comment:
Shouldn't those constants be moved to the store.rs module?

Reply:
Done.

Reply:
https://github.com/Liberum-Neto/liberum-neto/commit/b8a0902ccf38d9d357c77bf6730cb027c9e6c7d4